mungegithub/build-cop/emergency.sh

```diff
@@ -22,11 +22,12 @@ STOP=${1:-"stop"}
 PORT=${PORT:-9999}
 
 SELF_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=/dev/null
```

Comment:
eek, would rather eat the warning than litter code with this

Reply:
should we even have these scripts anymore? there is no build cop?

Reply:
We can probably just disable this warning entirely, for the WIP kubernetes/kubernetes shellcheck lint I've disabled the non-constant source warning at the "we're calling shellcheck" level because it doesn't actually seem to be an issue and the codebase uses this very liberally.